dandischema/models.py

```diff
@@ -1048,7 +1048,7 @@ class Resource(DandiBaseModel):
     relation: RelationType = Field(
         title="Resource relation",
         description="Indicates how the resource is related to the dataset. "
-        "This relation should satisfy: dandiset <relation> resource.",
+        r"This relation should satisfy: dandiset \<relation\> resource.",
```

Comment:
I agree with @satra that this is about rendering of the text in the UI and should be done in the UI level. Making modification here also, unfortunately, changes the text itself, not just its rendering.